bingosync-app/static/bingosync/style.css

```diff
@@ -84,6 +92,10 @@
     overflow-y: scroll;
 }
 
+.panel-body {
+    padding:8px;
```

Comment:
I'm guessing this is so that "Non-Lockout" doesn't break into two lines in the Room Settings panel?

If that's the only reason, it seems like up to 13px works too. Thoughts on changing this to 13px or even making the override specific to the Room Settings panel?